charts/linkerd2/templates/web.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         - -grafana-addr=linkerd-grafana.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:3000
         {{- end}}
         {{- if .Values.tracing.enabled }}
-        - -jaeger-addr={{.Values.tracing.jaeger.name}}.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:16686
+        - -jaeger-addr=linkerd-jaeger.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:16686
```

Comment:
Sorry I can't recall how this works...  The drawback of moving the values to the addon's `values.yaml` file is that they're no longer visible in the linkerd2 charts like `.Values.global.namespace` here? If that's the case shouldn't we remove the option to set the Jaeger name given it's gonna be hard-coded here?

Reply:
Yep, That makes sense and we removed `.name` fields for grafana too for the same reason.